src/tests/ftest/datamover/dm_large_dir.yaml

```diff
@@ -21,8 +21,8 @@ server_config:
 pool:
   mode: 146
   name: daos_server
-  scm_size: 42000000000       # 42G
-  nvme_size: 700000000000     # 700G
+  scm_size: 95%
+  nvme_size: 90%
```

Comment:
For large_dir this is okay, but realistically the datasize of the large_dir test is very small.
```
50k files * 4096 bytes ~= 200MB
```
Some other tests use a similar dataset with a much smaller pool size:
https://github.com/daos-stack/daos/blob/9025d7cfef2f366557c95df5f3a20a49c62a0668/src/tests/ftest/datamover/dm_obj_large_posix.yaml#L26-L27
https://github.com/daos-stack/daos/blob/9025d7cfef2f366557c95df5f3a20a49c62a0668/src/tests/ftest/datamover/dm_serial_large_posix.yaml#L24-L25

Reply:
> For large_dir this is okay, but realistically the datasize of the large_dir test is very small.
> 
> ```
> 50k files * 4096 bytes ~= 200MB
> ```
> 
> Some other tests use a similar dataset with a much smaller pool size:
> 
> https://github.com/daos-stack/daos/blob/9025d7cfef2f366557c95df5f3a20a49c62a0668/src/tests/ftest/datamover/dm_obj_large_posix.yaml#L26-L27
> 
> 
> https://github.com/daos-stack/daos/blob/9025d7cfef2f366557c95df5f3a20a49c62a0668/src/tests/ftest/datamover/dm_serial_large_posix.yaml#L24-L25

@daltonbohning yeah but the pool being created in the test looked pretty large in dm_large_file.yaml.

Reply:
@daltonbohning yeah I see, could also just use smaller pool size like those tests I suppose.

Reply:
@dsikich For `large_file` it matters. But for these `large_dir` tests I don't think it really matters either way, as long as the pool size is reasonably large. Probably not necessary to push another commit, anyway.